llama_hub/wordlift/README.md

```diff
@@ -0,0 +1,63 @@
+# WordLift Reader
+
+The WordLift GraphQL Reader is a connector to fetch and transform data from a WordLift Knowledge Graph using your the WordLift Key. The connector provides a convenient way to load data from WordLift using a GraphQL query and transform it into a list of documents for further processing.
+
+## Usage
+To use the WordLift GraphQL Reader, follow the steps below:
+
+1. Set up the necessary configuration options, such as the API endpoint, headers, query, fields, and configuration options (make sure you have with you the [Wordlift Key](https://docs.wordlift.io/pages/key-concepts/#wordlift-key)).
+2. Create an instance of the `WordLiftGraphQLReader` class, passing in the configuration options.
+3. Use the `load_data` method to fetch and transform the data.
+4. Process the loaded documents as needed.
+
+Here's an example of how to use the WordLift GraphQL Reader:
+
+```python
+import json
+import logging
+from llama_index import GPTVectorStoreIndex
+from llama_index.readers.schema.base import Document
+from langchain.llms import OpenAI
+from langchain.chains.question_answering import load_qa_chain
+from base import WordLiftGraphQLReader
```

Comment:
would replace this with `from llama_hub.wordlift.base import WordLiftGraphQLReader`

Reply:
Resovled